tutorials/index.md

```diff
@@ -1,5 +1,13 @@
 # Tutorials
 
+These tutorials will help you get familiar with Chainblocks.
+
+## Setting up the workspace
+
+We will be using <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a>, although any text editor could work.
+
+You will also need a compiled version of Chainblocks to run these tutorials. Chainblocks can be <a href="https://docs.fragcolor.xyz/code/02.building-chainblocks/" target="_blank">built from the sources.</a>.
```

Comment:
No HTML

Reply:
Reverted to markdown link tag in latest commit.